libs/components/modals/testing/src/modules/confirm/confirm-harness.ts

```diff
@@ -72,24 +88,9 @@ export class SkyConfirmHarness extends SkyComponentHarness {
       throw new Error('Cannot get custom buttons for confirm of type OK.');
```

Comment:
Should probably match whatever is above. 'Cannot get custom buttons for non-custom confirm type.'